src/ResultPage.js

```diff
@@ -108,157 +112,113 @@ function ResultPage({ setUserPlace, countyData, isCountyDataLoaded }) {
+                        <InformationHeading>Maßnahmen</InformationHeading>
+                        <Explanation>
                             Seit dem 02.11.20 gelten in Deutschland die
                             folgenden einheitlichen Regelungen. Diese können in
                             Außnahmefällen regional verschärft werden, deshalb
                             informiere dich auch bei deinem Gesundheitsamt.
-                        </p>
-                        <ul className="measure-list">
+                        </Explanation>
+                        <MeasureList>
                             {measures.map((measure) => (
                                 <li key={measure.id}>
-                                    <h5>{measure.heading}</h5>
-                                    <p>{measure.text}</p>
+                                    <MeasureHeading>
+                                        {measure.heading}
+                                    </MeasureHeading>
+                                    <Explanation>{measure.text}</Explanation>
                                 </li>
                             ))}
-                        </ul>
-                    </section>
+                        </MeasureList>
+                    </Information>
                 </>
             )}
-        </ResultPageStyled>
+        </>
     )
 }
 
-const ResultPageStyled = styled.div`
-    h2 {
-        margin-bottom: 21px;
-        line-height: 1.2;
-        font-size: 32px;
-        color: var(--secondary-green);
-    }
-
-    h3 {
-        margin-bottom: 24px;
-        line-height: 1.5;
-        font-size: 26px;
-    }
-
-    h4 {
-        font-size: 26px;
-    }
+const Heading = styled.h2`
+    margin-bottom: 21px;
+    line-height: 1.2;
+    font-size: 2rem;
+    color: ${(props) =>
+        props.hotspot ? 'var(--secondary-red)' : 'var(--secondary-green)'};
+`
 
-    p {
-        font-size: 16px;
-        color: var(--gray);
-        margin: 20px 0;
-    }
+const SubHeading = styled.h3`
+    margin-bottom: 24px;
+    line-height: 1.5;
+    font-size: 1.625rem;
+`
 
-    footer {
-        padding: 0 20px;
-    }
+const InformationHeading = styled.h4`
+    font-size: 1.625rem;
+`
 
-    .result-wrapper {
-        padding-bottom: 20px;
-    }
+const Explanation = styled.p`
+    font-size: 1rem;
+    color: var(--gray);
+    margin: 20px 0;
+`
 
-    .county-class-red {
-        background: var(--primary-red);
-        color: var(--silver);
-    }
+const Navigation = styled.nav`
+    padding: 0 20px;
+`
 
-    .county-class-green {
-        background: var(--primary-green);
-        color: var(--silver);
-    }
+const Result = styled.section`
+    padding-bottom: 20px;
+    background: ${(props) =>
+        props.hotspot ? 'var(--primary-red)' : 'var(--primary-green)'};
+    color: var(--silver);
+`
 
-    .result-text {
-        padding: 0 30px;
-        margin-bottom: 105px;
-    }
+const ResultText = styled.div`
```

Comment:
Semantically ``ResultText`` sounds like a section or a paragraph. If it is a wrapper or just styled, I would indicate this in the name